modules/indexer/issues/dboptions.go

```diff
@@ -38,6 +38,12 @@ func ToSearchOptions(keyword string, opts *issues_model.IssuesOptions) *SearchOp
 		searchOpt.MilestoneIDs = opts.MilestoneIDs
 	}
 
+	if opts.ProjectID > 0 {
+		searchOpt.ProjectID = optional.Some(opts.ProjectID)
+	} else if opts.ProjectID == -1 { // FIXME: this is inconsistent from other places
```

Comment:
Didn't we have a constant for that?
`db.NoneID` or something like that?

Reply:
There is one `db.NoConditionID` which have a different meaning from here so I don't use that one to not confuse code readers.